app/actors/hyrax/actors/interpret_visibility_actor.rb

```diff
@@ -60,14 +60,14 @@ def sanitize_unrestricted_params
                                :visibility_after_embargo)
           end
 
-          def sanitize_embargo_params
+          def sanitize_lease_params
```

Comment:
Buh. Why'd you swap the method names?

Reply:
Look at the implementation.  It was doing the lease stuff.